app/test_main.py

```diff
@@ -1,3 +1,21 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_two_parameters_equal_to_zero() -> None:
+    assert (get_human_age(0, 0) == [0, 0])
+
+
+def test_one_parameter_equal_to_zero() -> None:
+    assert (get_human_age(0, 21) == [0, 1])
```

Comment:
The test case `test_one_parameter_equal_to_zero` is valid. Ensure that similar cases are not duplicated in other tests.